src/language/LanguageProvider.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { createContext, useContext } from 'react';
-import sv from 'language/sv';
+import en from 'language/en';
 
 import languages, { Language } from 'language';
 import Storage from 'storage';
 
-const LanguageContext = createContext(sv);
+const LanguageContext = createContext(en);
```

Comment:
This is not enough to change the default. You need to change the default setting in storage as well.